src/Directory.Build.targets

```diff
@@ -15,6 +14,13 @@
                     RelativePath="$(IncludeInPackage)\lib\$(TargetFramework)" />
     </ItemGroup>
 
+    <ItemGroup Condition="'$(IncludeAnalyzerInPackage)' != ''">
+      <PackageAsset Include="$(TargetPath)"
+                    RelativePath="$(IncludeAnalyzerInPackage)\analyzers\dotnet\cs" />
```

Comment:
We may want to double check that we don't need this `dotnet` to be parallel with other tfms in the package to avoid the package installing and only supplying an analyzer on a unsupported framework.

Reply:
In other words, don't reintroduce this issue: https://github.com/dotnet/machinelearning/issues/357.

I think this should be simple to test, I'm not familiar enough with NuGet conventions for analyzers to know if it would have this problem without testing it.

Reply:
This may be an unfounded concern.  I looked into how NuGet resolves these and its actually different than the rest of the content.   Essentially these appear to be ignored by restore / package applicability checks, and just resolved via a separate content selection algo at build time (not restore time https://github.com/NuGet/NuGet.BuildTasks/blob/640c8e13a9b7ab6e86264a296638fbf3cc016ad1/src/Microsoft.NuGet.Build.Tasks/ResolveNuGetPackageAssets.cs#L371).

Net result, ignore this comment.

Reply:
Cool!! Thanks for your attention though @ericstj